projects/ngx-sub-form/src/lib/ngx-sub-form-utils.ts

```diff
@@ -30,6 +31,8 @@ export type KeysWithType<T, V> = { [K in keyof T]: T[K] extends V ? K : never }[
 
 export type ArrayPropertyOf<T> = KeysWithType<T, Array<any>>;
 
+export type ArrayTypeOfPropertyOf<T, K extends keyof T = keyof T> = T[K] extends Array<infer U> ? U : never;
```

Comment:
This type will only support a `FormInterface` that only has one `Array`.

If the `FormInterface` has another property that is not an array, it'll become `never`.

This would restrict the fact someone may wants to create a `FormArray` as part of a form with other properties, without passing that `FormArray` to a dedicated sub form.

I think the latter is a good practice and I think I'd be glad to enforce that behaviour but I think we discussed that and said it would be better to let people do what they want and that it might also be helpful while refactoring a form to use ngx-sub-form.

You can verify what I'm saying on that small repro: https://kutt.it/WqMoOv

Reply:
Huh good catch, fixed